jobtracker.go

```diff
@@ -352,12 +352,14 @@ func (jt *jobTracker) saveJob(job *job) {
 func (jt *jobTracker) updateJob(job *job) error {
 	details, err := jt.fetchJobDetails(job.Details.ID)
 	if err != nil {
+		metrics.Incr("updateJob.error", []string{"fetchJobDetails"}, 1)
```

Comment:
Nite: I notice you emit errors and successes as separate metrics. This is great, but it means that you have to write a success rate formula as `(successes / (successes + errors)) * 100`. It's often convenient to have a "total counter" that covers both successes and failures such that you remove the `successes + errors` bit. Also, errors tend to be sparse — if things are working well — and that sometimes causes display interpolation that confuses people and monitors. Iow, having a "total" emitted elsewhere might be a good idea!

Reply:
Also, you might want to put a tag key on these tags? Having `"error_type:fetchJobDetails"` means that you can group by `error_type` and get a pretty chart out of it.